main.py

```diff
@@ -107,7 +110,7 @@ async def health_check():
 
 
 @app.get("/documents", response_model=list[DocumentResponse])
-async def get_documents_by_ids(ids: list[str]):
+async def get_documents_by_ids(ids: list[str] = Query(...)):
```

Comment:
This is the essential change of this PR. The rest are just code formatting.

Without this change, even with the latest docker image https://github.com/danny-avila/rag_api/pkgs/container/librechat-rag-api-dev-lite/201676078?tag=8a7bbc2d0ae9291615f27708a196d3f7a2f2202d, I am getting the following error when running the 

```
╰─❯ curl --location 'http://localhost:8000/documents?ids=96f668f1-4f44-4d11-996d-9d6aa8245b05' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiJ9.e30.W6dlfFHLB_sIR942DvmGQeU9Bhh9HOECCzlsMWpD5Us'
{"detail":[{"type":"missing","loc":["body"],"msg":"Field required","input":null,"url":"https://errors.pydantic.dev/2.6/v/missing"}]}%   
```

Reply:
@jinzishuai I don't use anything special to format as far as I know, maybe autopep8 since it's default? Which do you use, I don't mind switching.

Reply:
we can settle on `black` https://pypi.org/project/black/

I can write a pre-commit hook for this

Reply:
Added this to the README:

### Dev notes:

#### Installing pre-commit formatter

Run the following commands to install pre-commit formatter, which uses [black](https://github.com/psf/black) code formatter:

```bash
pip install pre-commit
pre-commit install
```

Reply:
Try running the black formatter. You can use pre-commit or simply use it through your IDE

Reply:
perfect. I use black as well. I'll recreate this PR.